tests/functional/tests/rgw/test_rgw_tuning.py

```diff
@@ -0,0 +1,23 @@
+
+import pytest
+
+
+class TestRGWs(object):
+
+    def test_rgw_bucket_default_quota_is_set(self, node, File):
+        assert File(node["conf_path"]).contains("rgw override bucket index max shards")
+        assert File(node["conf_path"]).contains("rgw bucket default quota max objects")
+
+class TestMons(object):
+
+    def test_rgw_tuning_pools_are_set(self, node, Command):
+        cmd = "sudo ceph --cluster={} --connect-timeout 5 osd dump".format(node["cluster_name"])
+        output = Command.check_output(cmd)
+        pool1 = "foo"
```

Comment:
You could try pulling these values from ``node['vars']`` to avoid the hardcoding. It should have every config option ansible is aware of.